src/set_theory/game/pgame.lean

```diff
@@ -660,6 +660,102 @@ def relabel_relabelling {x : pgame} {xl' xr'} (el : x.left_moves ≃ xl') (er :
   relabelling x (relabel el er) :=
 relabelling.mk el er (λ i, by simp) (λ j, by simp)
 
+/-- `is_option' x y` means that `x` is either a either a left or right option of `y`, up to
+relabelling. Contrast with `is_option`. -/
+@[mk_iff] inductive is_option' : pgame → pgame → Prop
```

Comment:
```suggestion
@[mk_iff is_option'_iff_left] inductive is_option' : pgame → pgame → Prop
```
then you can call the other lemma below `is_option'_iff_right`, which keeps things nice and symmetric